examples/gallery/lucid_toy_car.py

```diff
@@ -57,10 +58,11 @@
 # Download the toy car geometry file (.fmd file exported by SpaceClaim).
 # Import geometry and display everything except tunnel.
 
+
 # For Windows OS users scdoc is also available:
-# toy_car = prime.examples.download_toy_car_scdoc()
+# toy_car = prime.examples.get_file(Examples.TOY_CAR_SCDOC)
 
-toy_car = prime.examples.download_toy_car_fmd()
+toy_car = prime.examples.get_file(Examples.TOY_CAR_PMDAT)
```

Comment:
@AlejandroFernandezLuces 
```suggestion
toy_car = prime.examples.get_file(Examples.TOY_CAR_FMD)
```